src/main/java/frc/robot/RobotContainer.java

```diff
@@ -8,16 +8,22 @@
 
 import edu.wpi.first.wpilibj.GenericHID;
 import edu.wpi.first.wpilibj.XboxController;
+import edu.wpi.first.wpilibj.GenericHID.Hand;
```

Comment:
```suggestion
import edu.wpi.first.wpilibj.GenericHID.Hand;
import edu.wpi.first.wpilibj.XboxController;
```
reorder to be alphabetical